brille/design/Proposed_integrated_syntax_Brill_SpinW_Horace.md

```diff
@@ -0,0 +1,275 @@
+and [the original](http://mslice.isis.rl.ac.uk/Main_Page) written in Matlab.
+The original Matlab MSlice is mostly a GUI program with an undocumented CLI (by using the underlying functions of the GUI).
+This program is also currently not being maintained.
+
+In Python, the syntax to extract data from a powder measurement using Mantid MSlice is:
+
+```python
+import mslice.cli as m
+dat2 = m.Load(nxs_file)
+cut2 = m.Slice(d2, f'|Q|,{qmin},{qmax},{qstep}', f'DeltaE,{emin},{emax},{estep}')
+```
+
+Note that unlike in Horace, the original pixel information is not retained in this rebinning operation.
+
+We propose the following syntax for powder data handling in Horace, should it be implemented:
+
+```matlab
+proj = projaxes('powder');
+cut2 = h.cut_sqw(sqw_file, proj, [qmin, qstep, qmax], [emin, estep, emax]);
+```
+
+This syntax should retain pixel information just like the equivalent single crystal invocation.
+
+
+## Modelling the data
+
+We assume that [SpinW](https://spinw.org/) would be used to model the magnetic excitations.
+To set up and evaluate a model on single crystal data with current syntax in Matlab is:
+
+```matlab
+tri = sw_model('triAF', 1); 
+spinw_pars = {'hermit', false, 'formfact', true, 'resfun', 'gauss'}; 
+sim1 = disp2sqw_eval(cut1, @tri.horace_sqw, spinw_pars, fwhm);
+```
+
+In Python it is:
+
+```python
+tri = h.sw_model('triAF', 1)
+spinw_pars = ['hermit', False, 'formfact', True, 'resfun', 'gauss']
+sim1 = h.disp2sqw_eval(cut1, tri.horace_sqw, fwhm, *spinw_pars)
+```
+
+As per [ADR #9](../../documentation/adr/0009-brille-integration.md), Brille is to be integrated within SpinW,
+which would permit the Matlab syntax:
+
+```matlab
+tri = sw_model('triAF', 1); 
+spinw_pars = {'hermit', false, 'formfact', true, 'resfun', 'gauss', ...
+              'use_brille', true, 'node_fraction_volume', 1e-5}; 
+sim1 = disp2sqw_eval(cut1, @tri.horace_sqw, spinw_pars, fwhm);
+```
+
+And in Python:
+
+```python
+tri = h.sw_model('triAF', 1)
+spinw_pars = ['hermit', False, 'formfact', True, 'resfun', 'gauss',
+              'use_brille', true, 'node_fraction_volume', 1e-5]
+sim1 = h.disp2sqw_eval(cut1, tri.horace_sqw, fwhm, *spinw_pars)
+```
+
+Whilst these syntax currently work, their implementation is still in review and have yet to be fully tested or released.
+The Python syntax also assumes that the Horace python distribution (`pyHorace`) will include SpinW as well as Horace.
+This would enable both to use the same Matlab interpreter process, and make intercommunication much easier and faster.
+
+Again these syntax should work for powder data objects, possibly replacing `h.disp2sqw`
+with a (yet to be implemented) `m.disp2sqw` method of `MSlice`.
+
+
+## Fitting the data with resolution convolution
+
+Horace includes a fitting framework which uses a Levenberg-Marquardt minimizer.
+The following example uses Brille and the TobyFit Monte Carlo resolution convolution method:
```

Comment:
Maybe add "although other fitting methods also use the framework" to clarify it isn't just Tobyfit